export/tests/print/test_service.py

```diff
@@ -115,29 +117,31 @@ def test_get_bad_printer_uri(self, mocked_call, capsys, mocker):
 
         assert ex.value.sdstatus is Status.ERROR_PRINTER_NOT_FOUND
 
-    @pytest.mark.parametrize(
-        "open_office_paths",
-        [
-            "/tmp/whatver/thisisadoc.doc"
-            "/home/user/Downloads/thisisadoc.xlsx"
-            "/home/user/Downloads/file.odt"
-            "/tmp/tmpJf83j9/secret.pptx"
-        ],
-    )
-    def test_is_open_office_file(self, capsys, open_office_paths):
-        assert self.service._is_open_office_file(open_office_paths)
-
-    @pytest.mark.parametrize(
-        "open_office_paths",
-        [
-            "/tmp/whatver/thisisadoc.doccc"
-            "/home/user/Downloads/thisisa.xlsx.zip"
-            "/home/user/Downloads/file.odz"
-            "/tmp/tmpJf83j9/secret.gpg"
-        ],
+    @mock.patch(
+        "subprocess.check_output",
+        return_value=b"/path/to/I Have Spaces.odg: application/vnd.oasis.opendocument.graphics\n",
     )
-    def test_is_not_open_office_file(self, capsys, open_office_paths):
-        assert not self.service._is_open_office_file(open_office_paths)
+    def test_needs_pdf_conversion(self, capsys):
+        file = tempfile.NamedTemporaryFile()
+
+        assert self.service._needs_pdf_conversion(file)
+
+    def test__get_supported_mimetypes_libreoffice(self, capsys):
+        """
+        Given well-formatted libreoffice sample .desktop files, assert correctly parsed.
+        """
+        # sample .desktop files in test directory, but the service checks /usr/share/applications
+        testdir_libreoffice_desktop = Path(Path.cwd(), "tests", "files")
+        supported = self.service._get_supported_mimetypes_libreoffice(testdir_libreoffice_desktop)
+        assert len(supported) > 0, len(supported)
```

Comment:
Since we're testing against fixed sample files, can we assert the real number? Or even just the full list of mime types?